### PR TITLE
[res/tfl_recipes] Add REGRESS_Issue_13863

### PR DESCRIPTION
**Pull request by Hanjin-Choi**
This commit adds REGRESS_Issue_13863 tflite recipe.

Related : https://github.com/Samsung/ONE/issues/13863
Draft : https://github.com/Samsung/ONE/pull/14051

ONE-DCO-1.0-Signed-off-by: HanJin Choi hanjin4647@gmail.com

### CODE REVIEW

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 1]
```diff
@@ -0,0 +1,252 @@
+operand {
+  name: "serving_default_input:0"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 32
+    dim: 32
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:7]: if not must use `32` plz use smaller single digit value.

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    quantized_dimension: 0
+  }
+  is_variable: false
+}
+operand {
+  name: "model/flatten/Reshape"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 3072
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:93]: ditto

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    quantized_dimension: 0
+  }
+  is_variable: false
+}
+operand {
+  name: "PartitionedCall:0"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 1024
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:105]: maybe ditto?

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+    quantized_dimension: 0
+  }
+  is_variable: false
+}
+operand {
+  name: "model/tf.split/split"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 1024
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:117]: maybe ditto

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+    quantized_dimension: 0
+  }
+  is_variable: false
+}
+operand {
+  name: "model/tf.split/split1"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 1024
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:129]: maybe ditto

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+    quantized_dimension: 0
+  }
+  is_variable: false
+}
+operand {
+  name: "model/tf.compat.v1.math.scalar_mul_1/Mul"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 3072
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:141]: maybe ditto

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+  }
+  is_variable: false
+}
+operand {
+  name: "PartitionedCall:2"
+  type: FLOAT32
+  shape {
+    dim: 1
+    dim: 1
+    dim: 3072
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:154]: maybe ditto...
and below values too

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 8]
```diff
@@ -0,0 +1,252 @@
+operand {
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:1]: if this is generated from tflchef-reverse, please add a comment from and how this was generated.

Comment (Hanjin-Choi) [reply res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe:5]: I changed recipe to single digit value and also add a comment how this recipe was generated.
Please take a look =)

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

File: res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe [hunk 9]
```diff
@@ -0,0 +1,256 @@
+# This recipe was created using tflchef-reverse with badDead.zip from the How_to_reproduce section of Issue_13863.
+# In the model, the dim value was changed to a single digit value, and the shape_signature was removed.
```
[the remark this answers is not in the document]
Comment (Hanjin-Choi) [reply res/TensorFlowLiteRecipes/REGRESS_Issue_13863/test.recipe]: I applied it according to your suggestion.

**Review by jinevening — APPROVED** [review]
LGTM

**Review by shs-park — APPROVED** [review]
LGTM!
=)

**Pull request merged**